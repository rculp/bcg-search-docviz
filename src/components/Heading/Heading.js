import { Header } from 'semantic-ui-react';
import './Heading.scss';

const Heading = Header;

export default Heading;
