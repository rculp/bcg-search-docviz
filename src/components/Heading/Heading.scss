@import 'src/global-styles/typography';

h1 {
  @extend %heading-large-sans;
}

h2 {
  @extend %heading-regular-sans;
}

h3 {
  @extend %heading-small-sans;
}