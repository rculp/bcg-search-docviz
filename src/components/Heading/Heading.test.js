import { shallow } from 'enzyme';

import React from 'react';

import Heading from './Heading';

describe('Heading', () => {
  it('renders and matches our snapshot', () => {
    const component = shallow(
      <Heading />
    );
    expect(component).toMatchSnapshot();
  });
});