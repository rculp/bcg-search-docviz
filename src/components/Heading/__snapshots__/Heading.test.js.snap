// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Heading renders and matches our snapshot 1`] = `
<div
  className="ui header"
/>
`;
